metabase/metabase
# Do not allow dashboard edit on small screens

Closes https://github.com/metabase/metabase/issues/23962
[Decision made](https://metaboat.slack.com/archives/C057T1QTB3L/p1689971581589849)

### Description

Do not allow dashboard edit on small screens. Overall dashboard editing is not usable on mobile and never intended to be. 

### How to verify

- Go to dashboard with filters using mobile phone or emulator with a small screen
- make sure pencil icon to edit dashboard is not visible
- 
### Demo

<img width="505" alt="image" src="https://github.com/metabase/metabase/assets/125459446/37760b13-21db-4db3-90a1-ed1abb464db0">


### Checklist
